COC.md

```diff
@@ -0,0 +1,43 @@
+# Code de conduite
+
+Ce code de conduite s'applique à tous les membres du Slack FranceJS.
+
+## Introduction
+
+La diversité et l'inclusion font la force de notre communauté.
+
+Nous encourageons la participation de tous et toutes et souhaitons être très clairs sur notre position.
+
+Notre objectif est de maintenir une communauté sûre, utile et amicale pour tous, sans distinction d'expérience, d'identité sexuelle et d'expression, d'orientation sexuelle, de handicap, d'apparence personnelle, de taille corporelle, de race, d'origine ethnique, d'âge, de religion ou de nationalité.
```

Comment:
"gender identity and expression" en fait "gender" est à la fois appliqué à "identity" et à "expression" (un peu comme si le mot avait été factorisé :p ), il s'agirait plutôt d'un truc comme "identité et expression du genre", "choix de genre et de la façon de l'exprimer", un truc comme ça. Pas simple à traduire ^^' ce qui est sûr c'est que cette portion doit porter sur le genre et non le sexe dans tous les cas.

Reply:
"sans distinction de genre, d'orientation sexuelle, de handicap..." ?

Reply:
vendu !